services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -52,3 +59,35 @@ func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
 		klog.Errorf("failed write data to response writer: %v", err)
 	}
 }
+
+func getQuotaInGib(r *http.Request) (*int, error) {
+	// When ContentLength is 0 that means request body is empty and
+	// storage quota is unlimited
+	if r.ContentLength == 0 {
+		return nil, nil
+	}
```

Comment:
This function should only handle a single concern: Covert a string containing a quota to a number

As such I expect to have a string passed into the function and not an HTTP request object.
This also means that the func should assume that a quota is codified in the string so the return value type is `int` and not `*int`

The handling of the request should be moved to the calling site and this fun should only be invoked if a  value was extracted from the request

Reply:
As the map creation moved to file scope, it doesn't make sense to have a function to read from the map, so I removed this function.